src/openrct2-ui/interface/Widget.cpp

```diff
@@ -413,7 +414,8 @@ static void widget_text(rct_drawpixelinfo* dpi, rct_window* w, rct_widgetindex w
         stringId = STR_STRING;
         formatArgs = &widget->string;
     }
-    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, l + 1, t, r - l);
+    //gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, l + 1, t, r - l);
+    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, coords, t);
```

Comment:
```suggestion
    gfx_draw_string_left_clipped(dpi, stringId, formatArgs, colour, {l + 1, r - 1}, t);
```
Create the coords in place as they're not used elsewhere (and remove them from above)